Documentation/free-surface-output.rst

```diff
@@ -29,3 +29,9 @@ variables
 
    | **u**, **v**, **w**: ground velocities, x y and z components
    | **U**, **V**, **W**: ground displacements, x y and z components
+
+Additionally, the writer outputs a quantity called "locationFlag", which has the values
+0 and 1 when at the elastic or acoustic side of an elastic-acoustic interface.
```

Comment:
Yes. The reason for this is that only the normal velocity need to be continuous - the other ones are discontinuous. So they actually have different values. This is also the reason for introducing the flags output. Otherwise we cannot distinguish between both sides.

Reply:
Can you add this explanation in the documentation? If I understand correctly, there will be several files then?

Reply:
No, there is only one file which contains all outputs. I'll make documentation a bit clearer.

Reply:
Thx